src/conf_mode/service_ipoe-server.py

```diff
@@ -88,22 +103,24 @@ def generate(ipoe):
     render(ipoe_conf, 'accel-ppp/ipoe.config.j2', ipoe)
 
     if dict_search('authentication.mode', ipoe) == 'local':
-        render(ipoe_chap_secrets, 'accel-ppp/chap-secrets.ipoe.j2',
-               ipoe, permission=0o640)
+        render(
+            ipoe_chap_secrets, 'accel-ppp/chap-secrets.ipoe.j2', ipoe, permission=0o640
+        )
     return None
 
 
 def apply(ipoe):
     systemd_service = 'accel-ppp@ipoe.service'
-    if ipoe == None:
+    if ipoe is None:
         call(f'systemctl stop {systemd_service}')
         for file in [ipoe_conf, ipoe_chap_secrets]:
             if os.path.exists(file):
                 os.unlink(file)
 
         return None
 
-    call(f'systemctl reload-or-restart {systemd_service}')
+    call(f'systemctl restart {systemd_service}')
```

Comment:
Why force a restart? `reload-or-restart` tries if daemon soft-reload is implemented and if not it will hard-restart the daemon.

Reply:
Accel-ppp does not do it correctly
The most of changes required restart is he service

Reply:
> Its restart doesn't interrupt user sessions, or does it?

It will drop user sessions